racer.el

```diff
@@ -231,7 +231,7 @@ foo(bar, |baz); -> foo|(bar, baz);"
   :lighter " racer"
   :keymap racer-mode-map
   (setq-local eldoc-documentation-function #'racer-eldoc)
-  (make-local-variable 'completion-at-point-functions)
+  (set (make-local-variable 'completion-at-point-functions) nil)
```

Comment:
Well, the plugin itself lists Emacs 24.3 as a dependency, so that doesn't sound like a problem unless that dependency doesn't do what I think it does, i.e. it doesn't tell people to install Emacs 24.3 in order to install the package.